Sources/Client/DeltaClientApp.swift

```diff
@@ -127,9 +132,17 @@ struct DeltaClientApp: App {
     Self.modalState.update(to: .error(message, safeState: safeState))
   }
   
-  /// Logs a fatal error and then fatal errors.
-  static func fatal(_ message: String) -> Never {
+  /// Logs a fatal error and redirects to troubleshooting page.
```

Comment:
The way you've done this, it looks like users would be able to exit the troubleshooting menu and keep using the app, which would defeat the purpose of fatal errors. Fatal errors are thrown when the app is not going to recover and is not going to be properly usable.

Reply:
You're right, fixing this 👍